src/client/index.js

```diff
@@ -326,14 +236,105 @@ const _sendMessage = (message) => {
   }
 }
 
+// React
+
+// Context to store session data globally
+const SessionContext = React.createContext()
+
+/** React hook to get the session */
+function useSession () {
+  return React.useContext(SessionContext)
+}
+
+/** Provider to wrap the app in to make session data available globally */
+function SessionProvider ({ children, session, options }) {
+  setOptions(options)
```

Comment:
Could/Should this be:
```suggestion
  React.useEffect(() => {setOptions(options)}, [])
```

Notice the empty dependency array. Do we expect options to change here?